### PR TITLE
Clean up more styles

### PR DESCRIPTION
**Pull request by devinjameson**

### CODE REVIEW

File: src/AffectedUserFlow/CodeInput/CodeInputScreen.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ const CodeInputScreen: FunctionComponent = () => {
     const enabledState: ENEnablement = "ENABLED"
     return exposureNotifications.status[1] === enabledState
   }
-  const isEnabled = hasExposureNotificationsEnabled()
+  const isEnabled = true
```
Comment (aledustet) [inline src/AffectedUserFlow/CodeInput/CodeInputScreen.tsx:17]: Should this be hardcoded?

File: src/AffectedUserFlow/CodeInput/CodeInputForm.tsx [hunk 2]
```diff
@@ -243,20 +243,21 @@ const style = StyleSheet.create({
...
     ...Buttons.secondary,
   },
   secondaryButtonText: {
-    ...Typography.buttonTextSecondary,
+    ...Typography.buttonSecondaryText,
   },
 })
 
 export default CodeInputForm
+
```
Comment (aledustet) [inline src/AffectedUserFlow/CodeInput/CodeInputForm.tsx:263]: I believe the linting will be sad about this

File: src/AffectedUserFlow/CodeInput/CodeInputScreen.tsx (hunk 1 above)
Comment (devinjameson) [reply src/AffectedUserFlow/CodeInput/CodeInputScreen.tsx:17]: Nope thanks!

File: src/AffectedUserFlow/CodeInput/CodeInputForm.tsx (hunk 2 above)
Comment (devinjameson) [reply src/AffectedUserFlow/CodeInput/CodeInputForm.tsx:263]: Yeah I always forget to format before throwing up PRs. 🦖

**Pull request merged**